sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 2.3.5-beta.1 (Unreleased)
+## 2.3.5 (2020-09-14)
```

Comment:
Do we not have any mentions?

Reply:
Hi @samvaity, actually we don't have any feature about `azure-spring-boot-starter-active-directory-b2c` in this version, but we need to unify the azure-spring-boot-starter version.
we have a hotfix before, so some of the spring-boot-starter version is 2.3.4, others are 2.3.3, so in this version, we unify the spring-boot-starter version to 2.3.5

![image](https://user-images.githubusercontent.com/8765864/93160064-24787e00-f742-11ea-8f27-e92db9f4fc68.png)